e2e/txserver/zeta_tx_server.go

```diff
@@ -156,6 +179,11 @@ func (zts ZetaTxServer) GetAccountName(index int) string {
 	return zts.name[index]
 }
 
+// GetCodec returns the codec for the ZetaTxServer
+func (zts ZetaTxServer) GetCodec() codec.Codec {
```

Comment:
should we switch to pointer semantic for all methods? Otherwise the struct is copied on each call
```suggestion
func (zts *ZetaTxServer) GetCodec() codec.Codec {
```

Reply:
aa8582481b6baf9f146439ea2a882a625b0b6219